app/assets/stylesheets/administrate/components/_buttons_group.scss

```diff
@@ -0,0 +1,95 @@
+// fixme please
+.buttons-group {
+  margin: $small-spacing;
+  padding-left: $small-spacing;
+  clear: both;
+}
+
+// copied from http://refills.bourbon.io/components/#er-toc-id-5
+.button-group {
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-line-height: 1.5em !default;
+  $base-spacing: 1.5em !default;
+  $base-font-size: 1em !default;
+  $base-background-color: white !default;
+  $action-color: #477DCA !default;
+  $dark-gray: #333 !default;
+  $large-screen: em(860) !default;
+  $base-font-color: $dark-gray !default;
+  $button-group-background: $base-background-color;
+  $button-group-color: lighten($base-font-color, 30%);
+  $button-group-border: 1px solid silver;
+  $button-group-inner-border: 1px solid lighten(silver, 18%);
```

Comment:
Color `silver` should be written in hexadecimal form as `#c0c0c0`<br>Color literals like `silver` should only be used in variable declarations; they should be referred to via variable everywhere else.